archlinuxcn/libgbinder/lilac.yaml

```
@@ -0,0 +1,19 @@
+maintainers:
+  - github: Puqns67
+
+build_prefix: extra-x86_64
+
+pre_build: vcs_update
```

[vcs_update](https://lilac.readthedocs.io/en/latest/api.html#lilac2.api.vcs_update) 是给 `-git` 包这种用的。
```suggestion
pre_build_script: update_pkgver_and_pkgrel(_G.newver)
```